rows.go

```diff
@@ -329,12 +336,11 @@ func (f *File) GetRowVisible(sheet string, row int) bool {
 //    xlsx.SetRowOutlineLevel("Sheet1", 2, 1)
 //
 func (f *File) SetRowOutlineLevel(sheet string, row int, level uint8) {
-	xlsx := f.workSheetReader(sheet)
 	if row < 1 {
-		return
+		panic(newInvalidRowNumberError(row)) // Fail fats to avoid possible future side effects!
```

Comment:
typo: `Fail fast` instead of `Fail fats`

Reply:
Thank you. I have fixed this typo.